pkg/auth/dependency/principal/password/normalize_test.go

```diff
@@ -0,0 +1,93 @@
+package password
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+
+	"github.com/skygeario/skygear-server/pkg/core/config"
+)
+
+func TestLoginIDEmailNormalizar(t *testing.T) {
+	newTrue := func() *bool {
+		b := true
+		return &b
+	}
+	newFalse := func() *bool {
+		b := false
+		return &b
+	}
+	Convey("TestLoginIDEmailNormalizar", t, func() {
+		type Case struct {
+			Email           string
+			NormalizedEmail string
+		}
+		f := func(c Case, n *LoginIDEmailNormalizar) {
+			result, _ := n.Normalize(c.Email)
+			So(result, ShouldEqual, c.NormalizedEmail)
+		}
+
+		Convey("default setting", func() {
+			cases := []Case{
+				{"Faseng@Example.com", "faseng@example.com"},
```

Comment:
Add a test case for NFKC. I think the test case should be written using `[]byte{}` literal otherwise it is visually indistinguishable.